app/assets/javascripts/discourse/app/components/reviewable-item.js

```diff
@@ -173,6 +173,7 @@ export default Component.extend({
         `/review/${reviewable.id}/perform/${performableAction.server_action}?version=${version}`,
         {
           type: "PUT",
+          dataType: "json",
```

Comment:
This is the reason the error wasn't showing; it's because of the way we catch invalid models on the server:

https://github.com/discourse/discourse/blob/ed3ed0f3dd94d5652a95331ce8efad2d38fd23e3/app/controllers/application_controller.rb#L168-L174